docs/build/sdk/v2/selling-assets.md

```diff
@@ -0,0 +1,53 @@
+# Selling Assets
+
+Your beliefs around a market might change during the market life time and in
+this case you might want to sell your previously bought assets.
+
+Lets say you have new information around a market and have the intuition that
+the price of your asset will drop to a price below what it is currently at in
+the near future. In this case you would like to sell those assets when the price
+is still high.
+
+:::info
+
+The following code is a continuation of the same code used when buying assets in
+the [Making a Prediction Part](/docs/build/sdk/v2/making-a-prediction)
+
+**But note that we have to calculate the slippage for selling and not buying.**
+
+:::
+
+### Swapping Outcome Assets for ZTG(base asset)
+
+Here we swap the previously bought outcome asset `Yes` back into the pool in
+exchange for `ZTG` or other base asset on the pool.
```

Comment:
`ZTG` and `Yes` should probably not be emphasized here.

Reply:
Fixed